<launch>

  <arg name="raw_ring_markers" default="true"/>

  <include file="$(find love_maker)/launch/robustifier.launch">
    <arg name="maximum_distance" value="0.5"/>
    <arg name="minimum_detections" value="3"/>
    <arg name="raw_detection_topic" value="/ring_detections_raw"/>
    <arg name="detection_topic" value="/ring_detections"/>
    <arg name="marker_topic" value="/ring_markers"/>
    <arg name="publish_raw_markers" value="$(arg raw_ring_markers)"/>
  </include>

</launch>
